satellite/satellitedb/users.go

```diff
@@ -163,6 +169,24 @@ func userFromDBX(ctx context.Context, user *dbx.User) (_ *console.User, err erro
 		result.ShortName = *user.ShortName
 	}
 
+	if user.isProfessional {
```

Comment:
you don't need this check. You should just be able to check `if user.Position != nil` for example. If you follow my other comment, the field will be nil anyway if `isProfessional` is false, so the extra check is unnecessary.